crates/services/gas_price_service/src/v0/service.rs

```diff
@@ -141,13 +141,15 @@ where
     Metadata: MetadataStorage,
 {
     async fn run(&mut self, watcher: &mut StateWatcher) -> TaskNextAction {
+        tracing::debug!("Starting gas price service");
```

Comment:
We might not want to log a "sunny day" events on the `debug!` level. Please consider using `trace!`.

Reference: [this comment](https://github.com/FuelLabs/fuel-core/pull/2383#discussion_r1845238000) (and a couple of other comments in the linked PR).